src/frontend/js/bulk-actions.ts

```diff
@@ -103,4 +106,29 @@ window.addEventListener("load", () => {
     // Submit form and execute bulk action
     form.submit();
   }
+
+  function hasTranslation(): boolean {
+    // checks if at least one of the selected pages has a translation for the current language
+    let urlPath = window.location.pathname;
+    // remove trailing slash from url path
+    urlPath = urlPath.replace(/\/$/, '');
+    // extract current language slug from url path
+    let languageSlug = urlPath.substr(urlPath.lastIndexOf('/') + 1);
+    // retrieve all checked input elements
```

Comment:
This seems kind of hacky to me. Can you maybe add another `data`-attribute for the language slug?

Reply:
Ok, data attribute is added